Whoaverse/Whoaverse/Controllers/CommentController.cs

```diff
@@ -344,11 +344,11 @@ public ActionResult SubmitComment()
                 {
                     // notify comment author that his comment has been deleted by a moderator
                     MesssagingUtility.SendPrivateMessage(
-                        "Whoaverse",
+                        "Voat",
                         commentToDelete.Name,
                         "Your comment has been deleted by a moderator",
                         "Your [comment](/v/" + commentSubverse + "/comments/" + commentToDelete.MessageId + "/" + commentToDelete.Id + ") has been deleted by: " +
-                        "[" + User.Identity.Name + "](/u/" + User.Identity.Name + ")" + " on: " + DateTime.Now + "  " + Environment.NewLine +
+                        "[" + User.Identity.Name + "](/user/" + User.Identity.Name + ")" + " on: " + DateTime.Now + "  " + Environment.NewLine +
```

Comment:
I know both are supported, it's just that /user/ isn't affected by this
issue: https://github.com/voat/voat/issues/379

On 26 March 2015 at 22:58, Atko <notifications@github.com> wrote:

> In Whoaverse/Whoaverse/Controllers/CommentController.cs
> <https://github.com/voat/voat/pull/378#discussion_r27264338>:
>
> >                          commentToDelete.Name,
> >                          "Your comment has been deleted by a moderator",
> >                          "Your [comment](/v/" + commentSubverse + "/comments/" + commentToDelete.MessageId + "/" + commentToDelete.Id + ") has been deleted by: " +
> > -                        "[" + User.Identity.Name + "](/u/" + User.Identity.Name + ")" + " on: " + DateTime.Now + "  " + Environment.NewLine +
> > +                        "[" + User.Identity.Name + "](/user/" + User.Identity.Name + ")" + " on: " + DateTime.Now + "  " + Environment.NewLine +
>
> Both /u/ and /user/ are supported.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/voat/voat/pull/378/files#r27264338>.
>